src/inmanta/data/__init__.py

```diff
@@ -2245,6 +2257,18 @@ def to_dto(self) -> m.Environment:
             validator=convert_boolean,
             doc="Allow the server to compile the configuration model.",
         ),
+        AUTO_FULL_COMPILE: Setting(
+            name=AUTO_FULL_COMPILE,
+            default="",
+            typ="str",
+            validator=validate_cron,
+            doc=(
+                "Periodically run a full compile following a cron-like time-to-run specification, interpreted in UTC"
+                " (`min hour dom month dow` or macros). A compile will be requested at the scheduled time. The actual"
```

Comment:
I included it because I believe it's a standard part of cron that people might expect to work. If they then read in the documentation only `min hour dom month dow` I was afraid we'd give them a false impression. But I see now there is the opposite risk as well. I'll remove the macros reference and prefix with "e.g.": "(e.g. `min hour dom month dow`)".